README.md

```diff
@@ -46,4 +46,39 @@ Configuration guide is available at https://github.com/DeloitteDigitalAT/core-ba
 
 - Sonar
 
-- TBD
\ No newline at end of file
+- TBD
+
+## How to run services and db locally. Requires JDK 11+ and Docker
+1) Build containers
+```
+cd data-access-services
+mvn install
```

Comment:
`mvn install` is not needed anymore as it is part of the `./docker-build.sh`

Reply:
good to know, removed line 'mvn install'